tutorials/bidi-xhtml/index.zh-hans.html

```diff
@@ -0,0 +1,141 @@
+f.contributors = ''; // people providing useful contributions or feedback during review or at other times
+// also make sure that the lang attribute on the html tag is correct!
+
+// TRANSLATORS should fill in these assignments:
+f.translators = '薛富侨'; // translator(s) and their affiliation - a elements allowed, but use double quotes for attributes
+f.translatorContact=""; // please add email. This is not displayed, it allows the translation coordinator to contact you if needed in future.
+
+f.breadcrumb = 'direction';
+</script>
+<script src="index-data/translations.js"> </script>
+<script src="../../javascript/doc-structure/article-dt.js"> </script>
+<script src="../../javascript/boilerplate-text/boilerplate-zh-hans.js"> </script>
+<!--TRANSLATORS must change -en in the line just above to the subtag for their language! -->
+<script src="../../javascript/doc-structure/article-2022.js"> </script>
+<script src="../../javascript/articletoc-2022.js"></script>
+<link rel="stylesheet" href="../../style/article-2022.css" />
+<link rel="stylesheet" href="index-data/local.css" type="text/css" />
+<link rel="copyright" href="#copyright"/>
+</head>
+
+<body>
+<header>
+<nav id="mainNavigation"></nav><script>document.getElementById('mainNavigation').innerHTML = mainNavigation</script>
+
+<h1>使用阿拉伯语、希伯来语和其他从右至左书写的文字创建HTML页面（教程）</h1>
+</header>
+
+<div id="sidebarExtras">
+  <section>
+    <h2 id="whyread" class="notoc">你为什么应该读这篇文章？</h2>
+    <p>让双向文本正确渲染有时会让人感觉很难，但事实并非如此。如果你感到困惑或者还没有开始，本教程可以帮助你采用最佳方法来标记内容，并充分解释双向算法的工作原理，以便你更好地理解大多数问题的根本原因。我们还会讨论有关处理双向文本标记语言的方法的一些常见误解。</p>
+  </section>
+  <section id="objectives">
+    <h2 class="notoc">目标</h2>
+    <p>读完本教程你可以学习到：</p>
+    <ul>
+      <li>创建包含用阿拉伯字母、希伯来字母或其他从右至左书写的文字的HTML页面</li>
+      <li>学习Unicode双向文本算法的基础知识，以便理解为什么双向文本会这样工作，以及如何解决问题</li>
+      <li>将HTML5中的重要新增内容应用到包含双向文本的页面</li>
+      <li>考虑不使用标记语言的替代方案</li>
+    </ul>
+    <p>很多语言都使用从右至左书写的文字，包括阿拉伯语、希伯来语、普什图语、波斯语、信德语、叙利亚语、迪维希语、乌尔都语、意第绪语、维吾尔语、哈萨克语和柯尔克孜语等。</p>
+  </section>
+</div>
+
+
+  <div id="audience">
+    <div id="updateInfo"></div><script>document.getElementById('updateInfo').innerHTML = g.updated</script>
+        </div>
+  <p>本教程汇总了一些文章，帮助你了解在编写HTML和CSS时如何使用从右至左书写的文字和双向文本。</p>
+
+<div id="nutshell">
+  <h2 id="quicksummary">简言之</h2>
+  <p>如果页面是由右至左书写的，向<code class="kw" translate="no">html</code>标签添加<code class="kw" translate="no">dir</code>属性以设置页面的默认基方向（base direction）。只有在你需要更改基方向时，才需要在页面内的块级元素上使用<code class="kw" translate="no">dir</code>属性。</p>
+  <p>对于行内文本，将所有<a class="termref" href="/International/articles/inline-bidi-markup/#oppositedirection">相反方向的短语</a><a  class="termref" href="/International/articles/inline-bidi-markup/#tightwrap">紧紧包裹</a>在设置其基方向的标记中。</p>
+  <p>使用<code>dir=auto</code>自动设置表单控件、<code class="kw" translate="no">pre</code>元素或插入页面的文本的基方向。如果需要将表单输入的基方向信息传递到服务器，请使用 <code class="kw" translate="no">dirname</code>属性。</p>
```

Comment:
dir=auto should probably have the translate=no attribute, and i'd be inclined to add  class=kw  too

Reply:
Added.